src/core/utils/MultisigGraphUtils.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MultisigAccountGraphInfo } from '../../model/account/MultisigAccountGraphInfo';
+import { MultisigAccountInfo } from '../../model/account/MultisigAccountInfo';
+
+/**
+ * Hash utilities for SecretLock hashing
+ */
+export class MultisigGraphUtils {
+    public static parseObjectProperties(obj, parse): any {
```

Comment:
- missing docs
- `any` type?

what does this method do?

Reply:
It simply helps traverse the tree object and for the return type, changed it to void.